app/models/package_manager/cran.rb

```diff
@@ -25,7 +25,7 @@ def self.check_status_url(db_project)
     end
 
     def self.project_names
-      html = get_html("https://cran.r-project.org/web/packages/available_packages_by_date.html", request: {timeout: 5})
+      html = get_html("https://cran.r-project.org/web/packages/available_packages_by_date.html", request: { timeout: 5 })
```

Comment:
The build was failing because of these lint errors after I rebased, since it seems like `main` was failing on them. These changes are just for that fix and are totally unrelated.